config/environments/production.rb

```diff
@@ -53,7 +53,7 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment)
-  # config.active_job.queue_adapter     = :resque
+  config.active_job.queue_adapter     = :inline
```

Comment:
Since we are deploying on Heroku, I am not sure how `inline` works.. We might still need a worker process for it to work properly. Is that right?

Reply:
inline runs in the same process. every job becomes a `#perform_now`. I'm thinking since its a small use case, maybe don't need setup worker process for now. or should I?